alkira/resource_alkira_service_f5_lb.go

```diff
@@ -191,23 +191,22 @@ func resourceAlkiraF5LoadBalancer() *schema.Resource {
 						},
 						"deployment_option": {
 							Description: "The deployment option of the F5 LB instance," +
-								" can be one of `ONE_BOOT_LOCATION` or `TWO_BOOT_LOCATION`." +
+								" Only supports `TWO_BOOT_LOCATION` for now." +
```

Comment:
If this is the case, could we hide this field till it's becoming optional. Right now, it's truly no option. Just hide it and use `TWO_BOOT_LOCATION` all the time.